docs/metrics.md

```diff
@@ -27,4 +27,66 @@ in-progress runs.
 ### Average run time
 
 The pipeline average run time is the average of the pipeline's wall time
-computed over all successful runs. Failed and canceled runs are not included.
\ No newline at end of file
+computed over all successful runs. Failed and canceled runs are not included.
+
+## Custom metrics
+
+{% hint style="info" %}
+
+Custom metrics are an Enteprise feature. Get in touch at
+[support@sematic.dev](mailto:support@sematic.dev) to learn more.
+
+{% endhint %}
+
+In addition to tracking and visualizing inputs and outpus of all your Functions,
+Sematic lets you log timeseries metrics during the execution of your pipeline
+and surfaces them in the Dashboard in real-time.
+
+This feature can be used to display a loss curve during training or track the
+evolution of certain hyperparameters such as learning rate. This is a similar
+functionality to what tools like Tensoboard offer.
+
+To log a metric value, simply call the `log_metric` API as such:
+
+```python
+from sematic.ee.metrics import log_metric
+
+@sematic.func
+def train_model(...):
+    for epoch in epochs:
+        _train(...)
+        
+        log_metric("loss", loss)
+```
+
+The logged values are plotted in real-time in the Metrics tab of the Run detais page.
+
+![Real-time metrics](images/liveMetrics.png)
+
+### Metric types
+
+Following the [OpenTelemetry
+specification](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#point-kinds),
+metrics can be aggregated in different ways. Sematic supports the following two
+aggregation mode:
+
+* Gauge (default) – within a given aggregation bucket, values are averaged.
```

Comment:
I wonder how is "aggregation bucket" defined. Is it hardcoded? If so, how much is it? If not, can it be customized?

Reply:
By default it is not aggregated, until there are over 300 points. Then buckets are `time_range / 300` so that there are always at most 300 points.
In the future we may allow customizing it but for now it is hard-coded.

Reply:
@neutralino1 Sounds good. Shall we also mention that to users in the doc?